inspec.yml

```diff
@@ -4,6 +4,6 @@ maintainer: spaterson@chef.io,russell.seymour@turtlesystems.co.uk
 summary: This resource pack provides compliance resources for AWS.
 copyright: spaterson@chef.io,russell.seymour@turtlesystems.co.uk
 copyright_email: spaterson@chef.io,russell.seymour@turtlesystems.co.uk
-version: 1.76.0
+version: 1.76.1
```

Comment:
This will be updated by expeditor

Reply:
again this cam from main branch I updated It.